src/Agent.Listener/_project.json

```diff
@@ -31,18 +31,25 @@
     },
     
     "dependencies": {
-        "NETStandard.Library": "1.0.0-rc3-23829",
+        "NETStandard.Library": "1.5.0-rc3-24011-00",
         "Microsoft.VisualStudio.Services.Agent": { "target": "project", "version": "1.0.0-*" },
         "vss-api-netcore": "0.5.4-private",
-        "System.ServiceProcess.ServiceController": "4.1.0-rc3-23829",
-        "System.IO.FileSystem.AccessControl": "4.0.0-rc3-23829",
+        "System.ServiceProcess.ServiceController": "4.1.0-rc3-24011-00",
+        "System.IO.FileSystem.AccessControl": "4.0.0-rc3-24011-00",
         "Newtonsoft.Json": "7.0.1",
-        "System.Diagnostics.TraceSource": "4.0.0-rc3-23829"
+        "System.Diagnostics.TraceSource": "4.0.0-rc3-24011-00",
+        "System.Security.AccessControl": "4.0.0-rc3-24011-00"
     },
 
     "frameworks": {
         "dnxcore50": {
             "imports": "portable-net451+win8"
         }
+    },
+
+    "runtimes": {
+        "win7-x64": { },
+        "ubuntu.14.04-x64": { },
```

Comment:
It looks that this change indeed limits us to a particular version of OSX and Unix. I could not find documentation how to overcome this, but it looks other people had this concern: "There should be a shared value for the "all Unix" RID collection" https://github.com/dotnet/buildtools/issues/444